### PR TITLE
citations plus recommendations

### PR DESCRIPTION
**Pull request by stpeter**

### CODE REVIEW

File: BCP195bis/draft-ietf-uta-rfc7525bis.md [hunk 1]
```diff
@@ -218,13 +218,17 @@ Rationale: Today, SSLv2 is considered insecure {{!RFC6176}}.
...
 
-* Implementations SHOULD NOT negotiate TLS version 1.0 {{?RFC2246}}; the only exception is when no higher version is available in the negotiation.
+* Implementations MUST NOT negotiate TLS version 1.0 {{?RFC2246}}.
                <vspace blankLines='1'/>
-Rationale: TLS 1.0 (published in 1999) does not support many modern, strong cipher suites. In addition, TLS 1.0 lacks a per-record Initialization Vector (IV) for CBC-based cipher suites and does not warn against common padding errors.
+Rationale: TLS 1.0 (published in 1999) does not support many modern, strong cipher suites. In addition, TLS 1.0 lacks a per-record Initialization Vector (IV) for CBC-based cipher suites and does not warn against common padding errors. 
+               <vspace blankLines='1'/>
+NOTE: This recommendation has been changed from SHOULD NOT to MUST NOT on the assumption that {{?I-D.ietf-tls-oldversions-deprecate}} will be published as an RFC before this document.
```
Comment (yaronf) [inline BCP195bis/draft-ietf-uta-rfc7525bis.md:225]: I think this citation needs to be normative ("!" within braces). We can change it in the future is that document is blocked for some reason.

File: BCP195bis/draft-ietf-uta-rfc7525bis.md [hunk 2]
```diff
@@ -245,17 +249,21 @@ This BCP applies to TLS 1.2, 1.3 and to earlier versions. It is not safe for rea
...
+* Implementations MUST support and (unless a higher version is available) MUST prefer to negotiate DTLS version 1.2 {{!RFC6347}} 
 
-* Implementations MUST support and MUST prefer to negotiate DTLS version 1.2 {{!RFC6347}}.
 <vspace blankLines='1'/>
   Version 1.2 of DTLS correlates to version 1.2 of TLS (see above).
   (There is no version 1.1 of DTLS.)
   
-* TODO: will require DTLS 1.3 at the SHOULD level if published before this document.
-
+* Implementations SHOULD support and, if available, MUST prefer to negotiate DTLS version 1.3 as specified in {{?I-D.ietf-tls-dtls13}}.
```
Comment (yaronf) [inline BCP195bis/draft-ietf-uta-rfc7525bis.md:264]: Normative again. This one is "publication requested" so I'm optimistic.

File: BCP195bis/draft-ietf-uta-rfc7525bis.md [hunk 3]
```diff
@@ -716,8 +720,6 @@ Robert Sparks and Dave Waltermire provided helpful reviews on behalf of the Gene
 
 During IESG review, Richard Barnes, Alissa Cooper, Spencer Dawkins, Stephen Farrell, Barry Leiba, Kathleen Moriarty, and Pete Resnick provided comments that led to further improvements.
```
Comment (yaronf) [inline BCP195bis/draft-ietf-uta-rfc7525bis.md:721]: Please add a word about these acknowledgements applying to the previous version of the RFC.

**Comment by stpeter** [comment]
@yaronf I've addressed your feedback. Feel free to merge at your convenience.

**Pull request merged**